azure-pipelines.yml

```diff
@@ -123,9 +123,9 @@ jobs:
 
     - template: etc/ci/azure-win.yml
       parameters:
-          job_name: win2016
```

Comment:
IMHO going from 2016 to 2022 is too big of a version jump at once..... what is the oldest version available after win2016? use this one instead